### ISSUE TITLE
Matplotlib version parsing crashes import

### ISSUE THREAD

**Issue by jhtu** [MHKiT-Software/MHKiT-Python#344]
### Describe the bug:
If MHKiT happens to be installed in an environment with a version of Matplotlib what contains non-integer characters (e.g., version 3.9.1.post1), then MHKiT will crash on import because it tries to parse the version number by splitting it apart at the decimal places, then casting each part to an int. That leads to `ValueError: invalid literal for int() with base 10: 'post1'` from line 14 of `mhkit/wave/contours.py`.

### To Reproduce:
```
$ python3 -m venv mhkit_bug
$ source mhkit_bug/bin/activate
$ pip install mhkit matplotlib==3.9.1.post1
$ python3 -c "from mhkit import contours"
```

### Expected behavior:
It should be possible to import MHKiT without raising a ValueError.

**Comment by ssolson** [comment]
@jhtu thank you for your interest in MHKiT and bringing this to our attention. 

I have patched this in #345 which I will get a new release out as soon as the tests pass.

**Comment by ssolson** [comment]
Fixed in MHKiT v0.8.2 (#347 )
